### PR TITLE
*: Refactor dist-agg partial result data layout

### PR DESCRIPTION
**Pull request by shenli**
Make result more compact. @coocood @BusyJay 
For example, count() do not need value field and sum() do not need count field.
SQL:	select count(c1), sum(c2), avg(c3) from t;
Aggs:	count(c1), sum(c2), avg(c3)
Rows:
Before this PR:
         	groupKey1, count1, value1, count2, value2, count3, value3
                groupKey2, count1, value1, count2, value2, count3, value3
After this PR:
                groupKey1, count1, value2, count3, value3
                groupKey2, count1, value2, count3, value3

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
@zimulala PTAL

File: executor/aggregate_xapi_test.go [hunk 1]
```diff
@@ -52,11 +52,11 @@ func (s *testAggFuncSuite) TestXAPICount(c *C) {
 	// Return row:
 	// GroupKey, Count, Value
```
Comment (zimulala) [inline executor/aggregate_xapi_test.go]: No longer need `Value` here.

**Comment by zimulala** [comment]
LGTM

**Pull request merged**